easybuild/easyconfigs/s/SWIG/SWIG-4.0.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,21 @@
+name = 'SWIG'
+version = '4.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
SWIG should be att GCCcore level, see https://github.com/easybuilders/easybuild-easyconfigs/pull/7923